mongo/options/clientoptions.go

```diff
@@ -356,7 +356,7 @@ func (c *ClientOptions) SetAuth(auth Credential) *ClientOptions {
 //
 // 2. "zlib" - requires server version >= 3.6
 //
-// 3. "zstd" - requires driver version >= 1.2.0, server version >= 4.2, and cgo support to be enabled.
+// 3. "zstd" - requires server version >= 4.2, and driver version >= 1.2.0 (cgo), or driver version >= 1.3.0 (without cgo)
```

Comment:
This is assuming the change is not backported, but let me know if this isn't accurate

Reply:
Let's be a little more explicit. Something like `and driver version >= 1.2.0 with cgo support enabled or driver version >= 1.3.0 without cgo`.

Reply:
changed